data/keywords.yml

```diff
@@ -368,6 +402,12 @@
       title: Uniqueness
     - url: 'https://json-schema.org/draft/2020-12/json-schema-validation#name-uniqueitems'
       title: Draft 2020-12
+- name: format
+  vocabulary: [Format Annotation]
+  learnjsonschemalink: 'https://www.learnjsonschema.com/2020-12/format-annotation/format/'
+  links:
+    - url: 'https://json-schema.org/draft/2020-12/draft-bhutton-json-schema-validation-00#rfc.section.7.2.1'
+      title: Draft 2020-12
```

Comment:
Here in UI both formats are appearing same. Any fix for it ?

![image](https://github.com/user-attachments/assets/4e601b55-e14d-412e-8783-6ae4fe5c7c29)

Reply:
Oh, I see now. The reason I suggested to remove them was because they were pointing to the wrong thing. I should have told you to switch them, not to remove them. The one for the "Format Assertion" vocabulary should have the link to format assertion and the one for "Format Annotation" vocabulary should have the link to format annotation. They were backwards before.